cmake/modules/FindIso9660pp.cmake

```diff
@@ -15,14 +15,16 @@ if(PKG_CONFIG_FOUND)
   pkg_check_modules(PC_ISO9660 libiso9660>=2.1.0 QUIET)
 endif()
 
+find_package(Cdio)
+
 find_path(ISO9660PP_INCLUDE_DIR NAMES cdio++/iso9660.hpp
-                                PATHS ${PC_ISO9660PP_INCLUDEDIR})
+                                PATHS ${PC_ISO9660PP_INCLUDEDIR} ${CDIO_INCLUDE_DIR})
```

Comment:
not needed

Reply:
If `libcdio` where found, it makes sense to add know path to the list, as `PATHS` are just a hints for CMake search.

Reply:
Removed